Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
That is exactly the point that every person that forks the project should be able to enable travis also on their branch. Like this potential travis problems pop up before the pull request is opened. As you see it is already enabled on mine ;-) https://travis-ci.org/ruflin/gotestcover